database/buildSchema/10B_template_filter.sql

```diff
@@ -0,0 +1,17 @@
+-- template filter tables
+CREATE TABLE public.filter (
+    id serial PRIMARY KEY,
+    code varchar NOT NULL UNIQUE,
+    icon_color varchar DEFAULT 'blue',
+    icon varchar,
+    title varchar,
+    query jsonb,
+    user_role public.permission_policy_type
+);
```

Comment:
So this table could also be used to store any filter, right? So if we allowed users to "bookmark" certain filtering configs for their list, this is where we'd store them?

Reply:
yep, can add some more join tables to user etc..